app/src/main/java/edu/uc/kovaciad/slicetracker/ui/main/MainFragment.kt

```diff
@@ -12,16 +12,13 @@ import edu.uc.kovaciad.slicetracker.R
 
 class MainFragment : Fragment() {
 
-    companion object {
-        fun newInstance() = MainFragment()
-    }
-
     private lateinit var viewModel: MainViewModel
     // For whatever reason, it won't compile when I do lateinit for this, but will if I don't
 //    private lateinit var applicationViewModel: ApplicationViewModel
 
     override fun onCreateView(
-        inflater: LayoutInflater, container: ViewGroup?,
+        inflater: LayoutInflater,
+        container: ViewGroup?,
```

Comment:
Expanded function signature to comply with Kotlin best practices: https://kotlinlang.org/docs/coding-conventions.html#functions